src/components/Core.vue

```diff
@@ -32,7 +32,21 @@
       RightCol,
       SendAskMoney,
     },
-  })
+    computed: {
+      getModalState () {
+	return this.$store.state.isModalOpen
+      }
+    },
+    watch: {
+      getModalState () {
+        if(this.getModalState)
+          document.body.classList.add('is-clipped')
+        else
+          document.body.classList.remove('is-clipped')  
+	  }
+      }
+    },
+  )
   export default class Core extends Vue {}
 </script>
 
```

Comment:
Just curious, why did you choose to watch for the modal state in this component ?
Why not in the `App.vue` which is "closer" to the `<body>` tag and the mechanism really is app-wide ?